### ISSUE TITLE
buttons seem not to be working

### ISSUE THREAD

**Issue by GingerBear** [MuscularMustache/GithuBubba#8]
@MuscularMustache Gabe, do you noticed that some button seem not working anymore, like the toggle. we probably need to change the selectors. also the button's style is also gone

**Comment by MuscularMustache** [comment]
Yeah i noticed this the other day. I'll make an update either tonight or tomorrow
